docs/user-guide/api-mediation/configuration-at-tls.md

```diff
@@ -327,3 +308,294 @@ An error can occur if the [list of ciphers](#ciphers) does not match between the
 Review the supported TLS versions and ciphers used in both the client and the server.
+
+## Full example of AT-TLS configuration
+
+Review a full working example of an AT-TLS configuration file on z/OS, specifically used for defining secure communication between different services in a mainframe environment. All port values are examples.
+The example is commented for convenience. 
+<details>
+
+<summary>Click here to display the full AT-TLS configuration file.</summary>
+
+```pagent
+# Main inbound rule, all API ML core services have it defined.
+TTLSRule ApimlServerRule
+{
+  LocalAddr All
+  RemoteAddr All
+  LocalPortRange 7554-7559
+  Jobname ZWE*
+  Direction Inbound
+  TTLSGroupActionRef ServerGroupAction
+  TTLSEnvironmentActionRef ApimlServerEnvironmentAction
+  TTLSConnectionActionRef ApimlServerConnectionAction
+}
+
+# Example southbound service inbound rule
+TTLSRule ApimlDCServerRule
+{
+  LocalAddr All
+  RemoteAddr All
+  LocalPortRange 40030-40040 # Example service ports
+  Jobname ZWEDC*
+  Direction Inbound
+  TTLSGroupActionRef ServerGroupAction
+  TTLSEnvironmentActionRef ApimlDCServerEnvironmentAction
+  TTLSConnectionActionRef ApimlServerConnectionAction
+}
+
+TTLSGroupAction ServerGroupAction
+{
+  TTLSEnabled On
+}
+
+# Environment action for API ML core services
+TTLSEnvironmentAction ApimlServerEnvironmentAction
+{
+  HandshakeRole ServerWithClientAuth
+  EnvironmentUserInstance 0
+  TTLSEnvironmentAdvancedParmsRef ServerEnvironmentAdvParms
+  TTLSKeyringParmsRef ApimlKeyring
+  TTLSSignatureParmsRef TNESigParms
+}
+
+# Environment action for sample southbound service
+TTLSEnvironmentAction ApimlDCServerEnvironmentAction
+{
+  HandshakeRole Server
+  EnvironmentUserInstance 0
+  TTLSEnvironmentAdvancedParmsRef ServerEnvironmentAdvParms
+  TTLSKeyringParmsRef ApimlKeyring
+  TTLSSignatureParmsRef TNESigParms
+}
+
+# Keyring, used for TLS
+TTLSKeyringParms ApimlKeyring
+{
+  Keyring ZWEKRNG
+}
+
+# Advanced TLS settings, choose TLS versions supported.
+# ClientAuthType Full to support optional x509 client cert authentication
+TTLSEnvironmentAdvancedParms ServerEnvironmentAdvParms
+{
+  ClientAuthType Full
+  ApplicationControlled Off
+  Renegotiation Disabled
+  SSLv2 Off
+  SSLv3 Off
+  TLSv1 Off
+  TLSv1.1 Off
+  TLSv1.2 On
+  TLSv1.3 On
+}
+
+# Server Connection Action for API ML core services.
+# Needs ServerWithClientAuth to support client certificate authentication between them.
+TTLSConnectionAction ApimlServerConnectionAction
+{
+  HandshakeRole ServerWithClientAuth
+  TTLSCipherParmsRef CipherParms
+  TTLSConnectionAdvancedParmsRef ApimlConnectionAdvParms
+  TTLSSignatureParmsRef TNESigParms
+}
+
+# API ML Server connection action.
+# Certificate label indicates which certificate is used in the client certificate authentication process between core services.
+TTLSConnectionAdvancedParms ApimlConnectionAdvParms
+{
+  ApplicationControlled Off
+  ServerCertificateLabel apimlcert
+  CertificateLabel apimlcert
```

Comment:
There is definitely missing a rule with empty string (to call a service, and other communication than again DS)

Reply:
Can you share the example rule?